municipal_finance/static/javascript/table.js

```diff
@@ -539,10 +551,10 @@
         // copy this, we're going to change it
         params.drilldown = params.drilldown.slice();
 
-        // ensure the download has all attributes, except function
-        // which we'll deal with later
+        // ensure the download has all relevant attributes.
+        // we only include functions if we're already filtering on functions
         _.each(cube.model.dimensions, function(dim, dim_name) {
-          if (dim_name != 'function') {
+          if (dim_name != 'function' || hasFunctions) {
```

Comment:
generally I try to refer to govt. functions when space allows else I end up thinking we're dealing with higher order functions for a while, but that can become verbose